packages/frontend-usagers/src/components/personnes.vue

```diff
@@ -84,7 +88,11 @@ const personnesToDisplay = computed(() => {
   return props.personnes.map((p, index) => {
     const row = [];
     displayedFields.forEach((f) => {
-      row.push(Array.isArray(p[f]) ? p[f].join(",") : p[f]);
+      let value = Array.isArray(p[f]) ? p[f].join(",") : p[f];
```

Comment:
```suggestion
      let valueFormatted = Array.isArray(personne[field]) ? personne[field].join(",") : personne[field];
```
il faudrait que le nom des variables soit intelligible pour faciliter la lecture.
sinon le nom "value" est un peu trop générique, je te propose `valueFormatted` qui est plus clair.
et si besoin on peut faire une variable intermédiaire `const fieldValue = personne[field]`